examples/hello/src/operations.ts

```diff
@@ -18,4 +26,10 @@ export class Hello {
     const greet_count = rows[0].greet_count;
     return `Hello, ${user}! You have been greeted ${greet_count} times.\n`;
   }
+
+  @OperonInitializer()
```

Comment:
I like this interface!  But what happens if you have multiple @OperonInitializer?

Reply:
I would like to split it into 2 decorators - One for Init and one for Deploy.
We would call them both at the same time for dev and test... in production the distinction would be made.

Reply:
@chuck-dbos What do we mean by deploy ? I guess for our cloud - we know what that might mean ? But how to enforce is in other deployment environments.

Reply:
In development or test, which is single node, every start is (potentially) a deployment, be it an initial or update deployment.  I am assuming the user code will figure out which it is.

Reply:
Current code does that. deploy decorator can be added later when we know for sure when/how/what about deploy